elasticizer/__init__.py

```diff
@@ -1 +1,2 @@
 from elasticizer.pipelines import *
+from elasticizer import __main__
```

Comment:
Is this line necessary?  It would change my understanding of __init__ and __main__